VirtoCommerce.CustomerModule.Data/Services/MemberServiceBase.cs

```diff
@@ -106,7 +106,9 @@ public virtual void SaveChanges(Member[] members)
                             if (dataTargetMember != null)
                             {
                                 changeTracker.Attach(dataTargetMember);
-                                changedEntries.Add(new GenericChangedEntry<Member>(member, dataTargetMember.ToModel(AbstractTypeFactory<Member>.TryCreateInstance(member.MemberType)), EntryState.Modified));
+                                var oldEntry = dataTargetMember.ToModel(AbstractTypeFactory<Member>.TryCreateInstance(member.MemberType));
```

Comment:
This one-by-one dynamic property loading can lead to performance degradation of saving members. I think will be better to used batch operation for `LoadDynamicPropertyValues(params IHasDynamicProperties[] owners)` for all `old` members out of this loop.